src/WebUI/Common/KubeSummary.tsx

```diff
@@ -178,7 +210,7 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
                 </TabBar>
                 <TabContent>
                     {this.state.selectedPivotKey === servicesPivotItemKey && <ServicesPivot kubeService={this.props.kubeService} namespace={this.state.namespace} filter={this.state.svcFilter} filterToggled={servicesFilterToggled} />}
-                    {this.state.selectedPivotKey === workloadsPivotItemKey && <WorkloadsPivot kubeService={this.props.kubeService} namespace={this.state.namespace} filter={this.state.workloadsFilter} filterToggled={workloadsFilterToggled} markTTI={this.props.markTTI} />}
+                    {this.state.selectedPivotKey === workloadsPivotItemKey && <WorkloadsPivot kubeService={this.props.kubeService} namespace={this.state.namespace} filter={this.state.workloadsFilter} filterToggled={workloadsFilterToggled} markTTI={this.props.markTTI} historyService={this.props.historyService} />}
```

Comment:
this.props.historyService --> do we have historyService in props?

Reply:
it was an old artifact. thanks for spotting. removed it